Makefile

```diff
@@ -9,6 +9,7 @@ export IMAGE_VERSION = v1.9.0
 export SIMPLE_VERSION = $(shell (test "$(shell git describe)" = "$(shell git describe --abbrev=0)" && echo $(shell git describe)) || echo $(shell git describe --abbrev=0)+git)
 export GIT_VERSION = $(shell git describe --dirty --tags --always)
 export GIT_COMMIT = $(shell git rev-parse HEAD)
+# TODO: bump this to 1.21, after kubectl `--generator` flag is removed from e2e tests.
```

Comment:
leaving this for now, since `--generator` flag is deprecated [here](https://github.com/operator-framework/operator-sdk/blob/35313e88b5508c17a0cbb592f4b832f1692f715e/test/e2e/go/cluster_test.go#L139-L146) and the test needs to be re-written, in sdk and [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder/blob/81349c951eb31f04720ee37c4bbdf99bcf21743a/test/e2e/v3/plugin_cluster_test.go#L287). Will do this in a follow up PR, as I didn't want to block updating of k8s in projects because of this.